### ISSUE TITLE
How to enable DLSS?

### ISSUE THREAD

**Issue by francescofugazzi** [NVlabs/instant-ngp#579]
Is there any way to enable DLSS on the new version?

**Comment by cpkcpkcpk** [comment]
I'm also confused about this. Is DLSS disabled by default?

**Comment by Tom94** [comment]
Yes, it is disabled by default. If you have Vulkan installed (and it is correctly detected by CMake), __instant-ngp__ is built with DLSS support. If you then start the GUI, it'll attempt to initialize Vulkan and NGX -- only if this step succeeds (check the console log) will there be a "DLSS" checkbox in the GUI that you can enable. Cheers!
